### PR TITLE
Rowan Dempster - CV Bootcamp

### PR DESCRIPTION
**Pull request by rowandempster**
I made a CNN and put in some data augmentation.

Results:
image

image

image

image

### CODE REVIEW

**Comment by rowandempster** [comment]
@Trotyl15 Any ideas what's going on with the `val_loss`?

**Comment by Trotyl15** [comment]
> @Trotyl15 Any ideas what's going on with the `val_loss`?
:0 I'm actually not that sure Lol, maybe @aydanjiwani , but I think as long as you get one of the configs to work, it's just fine.

**Comment by aydanjiwani** [comment]
my guess is that the learning rate is too high or something weird is going on in the preprocessing (unlikely). I would try playing around with different optimizers and not just using adam but the accuracy is fine tbh

**Comment by rowandempster** [comment]
I don't want this to become a time sink so I'm gonna leave my experimentation here, thanks for the bootcamp ya'll I enjoyed it!
